### PR TITLE
Update todo tsp

### PR DESCRIPTION
**Pull request by v-jiaodi**

### CODE REVIEW

File: packages/typespec-ts/src/modular/buildCodeModel.ts [hunk 1]
```diff
@@ -501,6 +501,10 @@ function emitBodyParameter(
       isBinaryPayload: isBinaryPayload(context, body.type, contentTypes),
       optional: !isBodyRequired(httpOperation.parameters)
     };
+  } else if (body.bodyKind === "multipart") {
```
Comment (qiaozha) [inline packages/typespec-ts/src/modular/buildCodeModel.ts:504]: I am not sure I follow the fixes here? we already filtered out the bodyKind for single type which is what we can handle in line 485.

File: packages/typespec-ts/src/modular/buildCodeModel.ts [hunk 2]
```diff
@@ -913,7 +918,10 @@ function emitBasicOperation(
     // Flatten the body parameter if it is an anonymous model
     const originalBodyType = httpOperation.parameters.body.type;
 
-    if (
+    // skip body parameter if it is not a body parameter
+    if (!bodyParameter) {
```
Comment (MaryGao) [inline packages/typespec-ts/src/modular/buildCodeModel.ts:922]: we crashed with undefined here.

File: packages/typespec-ts/src/modular/buildCodeModel.ts (hunk 1 above)
Comment (MaryGao) [reply packages/typespec-ts/src/modular/buildCodeModel.ts:504]: This is more for comment purpose. because this is where to handle mfd.

**Comment by qiaozha** [comment]
Can you verify if it works in latest main? if it has been resolved, can you close this pr?

**Comment by v-jiaodi** [comment]
Closed as it works in latest main.

**Pull request closed without merging**